apollo-router/src/orbiter/snapshots/apollo_router__orbiter__test__visit_config.snap

```diff
@@ -2,20 +2,18 @@
 source: apollo-router/src/orbiter/mod.rs
 expression: usage
 ---
-configuration.headers.all.request.insert.len: 1
 configuration.headers.all.request.insert.name.<redacted>: 1
 configuration.headers.all.request.insert.value.<redacted>: 1
-configuration.headers.all.request.propagate.len: 2
```

Comment:
why are we losing propagate and remove?

Reply:
I've improved it a bit: [e096eb1](https://github.com/apollographql/router/pull/5061/commits/e096eb19cf908c8a865b61b3312c181045e0ccd6)